README.md

```diff
@@ -34,19 +34,35 @@ A summary of the currently supported features:
 
 Grab a Phabricator token at `https://<yourserver.com>/settings/panel/apitokens/`
 
-Configure:
+Configure in Linux:
 
      export PHAB_TOKEN=cli-ABC123
     --OR--
      echo "PHAB_TOKEN: cli-ABC123" > ~/.config/phabfive.yaml
 
+Configure in Windows:
+    Phabfive looks for configuration in the following sequence in Windows environment:
+      %ALLUSERSPROFILE%\phabfive\phabfive.yaml
+      %ALLUSERSPROFILE%\phabfive\phabfive.d\*.yaml
+      %LOCALAPPDATA%\phabfive\phabfive.yaml
+      %LOCALAPPDATA%\phabfive\phabfive.d\*.yaml
```

Comment:
This block looks not good when viewed as a markdown file. Try to make this into a list with 4 items instea

Reply:
changed the structure - was not familiar with md - thanks!